corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -172,6 +172,19 @@
 
 }
 
+.case-tile-container {
```

Comment:
Nitpick: could you pull the `.case-tile-container` styles from `content.less` in here, so they're in one place?